lib/logger_json/formatter.ex

```diff
@@ -1,6 +1,7 @@
 defmodule LoggerJSON.Formatter do
   @type opts :: [
-          {:metadata, :all | {:all_except, [atom()]} | [atom()]}
+          {:encoder_opts, term()}
```

Comment:
@AndrewDryga wold it make sense to use `[Jason.encode_opt()]` instead of `term()`? I chose `term()` since we don't control `Jason` and not relying on `Jason.encode_opt()` makes the implementation less brittle. Jason is pretty stable so I imagine this isn't a huge concern but erring on the side of caution for now.

Reply:
Yeah, let's do `[Jason.encode_opt()]`, we have Jason as a hard dependency and maybe it will help Dialyzer to catch some user errors.

Reply:
Resolved by [c00876a](https://github.com/Nebo15/logger_json/pull/130/commits/c00876af405606a3f06290e6fa56cac272c2ba63)